test/e2e/specs/editor/various/pref-modal.spec.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Preferences modal', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test.describe( 'Preferences modal adaps to viewport', () => {
+		test( 'Enable pre-publish flow is visible on desktop ', async ( {
+			page,
+		} ) => {
+			await page.click(
```

Comment:
These `page.click` calls seem to be discouraged in the [Playwright Docs](https://playwright.dev/docs/api/class-page#page-click) (although I didn't see any reasoning in the docs, the note was added as [a follow-up to some deprecations](https://github.com/microsoft/playwright/pull/19025)).

Does it still work if you use `page.locator.click` instead?